src/status_im/chat/commands/protocol.cljs

```diff
@@ -68,3 +68,7 @@
     "Function which takes original parameters + cofx map and returns new map of parameters")
   (enhance-receive-parameters [this parameters cofx]
     "Function which takes original parameters + cofx map and returns new map of parameters"))
+
+(defprotocol Extension
```

Comment:
How about having some concept a bit more generic? Like `Context` that would allow to access a map, eventually containing `::extension/id` ?

Reply:
i'm not sure we need this , also we need to check a type if it's extension or not

Reply:
Ok maybe let's keep this for now. If we need more we can introduce something more generic later.